cla-backend-go/v2/signatures/handlers.go

```diff
@@ -1304,8 +1306,26 @@ func isUserHaveAccessToCLAGroupProjects(ctx context.Context, authUser *auth.User
 		return false
 	}
 	if len(projectCLAGroupModels) == 0 {
-		log.WithFields(f).Debug("no project cla group mappings by CLA Group ID - failed permission check")
-		return false
+		projectCLAGroup, err = projectRepo.GetCLAGroupByID(ctx, claGroupID, false)
+		if err != nil {
+			log.WithFields(f).WithError(err).Warnf("problem loading project cla group mappings by CLA Group ID - failed permission check")
```

Comment:
Let's update this comment to say: "problem loading cla group by ID - failed permission check"

Reply:
Fixed.